howto/install-appliance/finalise.md

```diff
@@ -0,0 +1,84 @@
+After installing the Anbox Cloud Appliance, you must initialise it and register your Ubuntu SSO account with the dashboard.
+
+<a name="initialise"></a>
+## Initialise the appliance
+
+Invoke the initialisation process of the Anbox Cloud Appliance by entering the following command on your machine:
+
+    sudo anbox-cloud-appliance init
+
+You will be asked a few questions. If you don't want to make any specific changes, you can safely stay with the offered default answers.
+
+```bash
+Welcome to the Anbox Cloud Appliance!
+
+The following questions will guide you through the initial setup of the
+appliance. If you don't care about answering any of them you can just
+accept the defaults.
+
+For any further questions please have a look a the official Anbox Cloud
+documentation at https://anbox-cloud.io/docs
+
```

Comment:
This needs a bit of update to honor the feature of multiple node support we added since 1.14 release

````
For any further questions please have a look a the official Anbox Cloud
documentation at https://anbox-cloud.io/docs

Are you joining this node to an existing Anbox Cloud Appliance installation? (EXPERIMENTAL) [default=no] 

Both the containers used to deploy the control plane services of the
Anbox Cloud Appliance and the ones used for the actual Android instances
```